chart/flux/templates/helm-operator-crd.yaml

```diff
@@ -107,6 +107,9 @@ spec:
             releaseName:
               type: string
               pattern: "^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"
+            timeout:
```

Comment:
Should this also be in the `HelmRelease` CRD?

Reply:
Sorry I misplaced the validation, I've moved it from FHR to HR in the latest commit.